bpa-services/src/main/java/org/egov/bpa/service/BPAService.java

```diff
@@ -323,12 +383,11 @@ public BPA update(BPARequest bpaRequest) {
 				bpa.getApplicationNo());
 
 		List<BPA> searchResult = getBPAWithBPAId(bpaRequest);
-		if (CollectionUtils.isEmpty(searchResult)) {
+		if (CollectionUtils.isEmpty(searchResult) || searchResult.size() > 1) {
```

Comment:
Shouldn't give same error when size is more than 1. Please provide a better in that case so it is relevant to caller